packages/shared/styles/global.ts

```diff
@@ -8,10 +8,22 @@ import {
 } from "../../design-tokens/build/js/designTokens";
 import { normalize } from "./normalize";
 
+// this is used to remove top and bottom margins for <p> and <h1-6> tags
+//
+// removing the margin directly in the <Text> component styles makes it impossible to use
+// child selectors to change the margins.
+// e.g.: using the Stack or TextBlock components.
+export const staticClass_resetTextMargin = "rmMargins";
```

Comment:
rmTextMargins?

Reply:
I like this suggestion. I'll make that small update and use admin permissions to merge ;)